testData/solutions/win32.json

```diff
@@ -12,13 +12,49 @@
             {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\enu\\DEFAULT.JCF",
```

Comment:
Hi @cstrobbe, sorry for the late reply. Yup, those are the two files. Default.JCF is the main settings file with all the settings. The VoiceProfiles.ini is just for the voice settings such as speech rate. From what I know, you can change the language settings in the VoiceProfile.